tests/benchmarks/cli/commands/test_init.py

```diff
@@ -1,12 +1,18 @@
 import os
 import shutil
+from contextlib import suppress
 
 
-def remove_dvc_root():
-    dvc_root = ".dvc"
-    if os.path.isdir(dvc_root):
-        shutil.rmtree(".dvc")
 
 
 def test_init(bench_dvc, tmp_dir, scm):
-    bench_dvc("init", setup=remove_dvc_root, rounds=10, warmup_rounds=1)
+    def _cleanup_dir():
+        for root, dirs, files in os.walk(tmp_dir):
```

Comment:
btw, tmp_dir is path-like, right? If so, you could probably use iterdir instead since you don't really walk in-depth there anyway.